.gitlab/pipeline/check.yml

```diff
@@ -4,8 +4,11 @@ cargo-clippy:
     - .docker-env
     - .common-refs
     - .pipeline-stopper-artifacts
+  variables:
+    RUSTFLAGS: "-Dwarnings"
   script:
-    - SKIP_WASM_BUILD=1 env -u RUSTFLAGS cargo clippy --all-targets --locked --workspace
+    # TODO: remove before merge - temporary allow exceptions
+    - SKIP_WASM_BUILD=1 cargo clippy --all-targets --locked --workspace -- --allow clippy::type_complexity
```

Comment:
```suggestion
```

Reply:
I think you mean just remove `--allow clippy::type_complexity` @ggwpez ?